opentelemetry-sdk/src/opentelemetry/sdk/_metrics/export/__init__.py

```diff
@@ -172,4 +174,5 @@ def _shutdown():
 
         self._shutdown_event.set()
         self._daemon_thread.join()
-        return self._exporter.shutdown()
```

Comment:
I am wondering if we should actually update the metric exporter return the status here.

Reply:
For what it's worth returning `None` is the same across the other signals' exporter interfaces.

@open-telemetry/python-approvers thoughts?

Reply:
From the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#shutdown-1).
> `Shutdown` SHOULD provide a way to let the caller know whether it succeeded, failed or timed out.

We need to return something else than a boolean value.

Reply:
@ocelotl I see that the linked spec is for `MetricReader` while component in question is about exporter which doesn't have any return type [spec'd](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#shutdown-2). For tracing/logging exporters we do clean up such as closing the connection and just return. I don't have any strong opinion about this.

Reply:
Ah, I see. Well, in any case if we are to implement a timeout mechanism for this it could be convenient to return something that is informative about this.